core/tests/waste-item/use_cases/test_get_objets_recycling_material.py

```diff
@@ -0,0 +1,51 @@
+import unittest
+from unittest.mock import MagicMock
+from core.app.waste_item.application.use_cases.get_objets_recycling_material import GetObjetsRecyclingMaterialUseCase
+from core.app.waste_item.domain.ports import WasteItemRepository
+from core.app.waste_item.application.stats_material_dto import StatsWasteItem
+
+class TestGetObjetsRecyclingMaterialUseCase(unittest.TestCase):
+    def setUp(self):
+        # Crear un mock del repositorio
+        self.mock_repository = MagicMock(spec=WasteItemRepository)
+        self.use_case = GetObjetsRecyclingMaterialUseCase(self.mock_repository)
+
+    def test_execute_returns_material_and_count(self):
+        # Configurar el mock para devolver un objeto StatsWasteItem
+        material_waste = "Plastic"
+        expected_item = StatsWasteItem(material=material_waste, count=10)
+        self.mock_repository.getMaterial.return_value = expected_item
+
+        # Ejecutar el caso de uso
+        result = self.use_case.execute(material_waste)
+
+        # Verificar que el repositorio fue llamado correctamente
+        self.mock_repository.getMaterial.assert_called_once_with(material_waste)
+
+        # Verificar que el resultado sea el esperado
+        self.assertEqual(result, {"material": "Plastic", "count": 10})
+
```

Comment:
remember, we are not using raw dictionaries, we should always use pydantic models
```suggestion
        self.assertEqual(result, StatsWasteItem(material="Plastic", count=10))
```